bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone.syphon",
-  "version": "0.6.3",
+  "version": "1.0.0",
```

Comment:
That said, if that's the direction we look to go, I think I should re-add support for jQuery 2.x as it's still widely used and supported.